specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json

```diff
@@ -0,0 +1,1585 @@
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "Indicates the version of the Queue service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/TableResponse"
+            }
+          },
+          "204": {
+            "description": "Success, table created.",
+            "headers": {
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "description": "This header uniquely identifies the request that was made and can be used for troubleshooting the request."
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "Indicates the version of the Queue service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageError"
+            }
+          }
+        }
+      }
+    },
+    "/Tables('{table}')": {
+      "delete": {
+        "tags": [
+          "table"
+        ],
+        "operationId": "Table_Delete",
+        "x-ms-examples": {
+          "TableDelete": {
+            "$ref": "./examples/TableDelete.json"
+          }
+        },
+        "description": "Operation permanently deletes the specified table",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/tableNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
In my testing, this call fails with a '415  Unsupported Media Type` error unless an `Accept` header is sent with 'application/json'. I confirmed that the Azure Data Explorer also sends this header on table deletes. Could we add a media type hint here?

Reply:
I added the content-type/accept values to this method - however they're not generated in Python so I can't guarantee that this will fix it. We may need to make alterations to the modeler/generators.